templates/organization/mission_type/index.html.twig

```diff
@@ -2,31 +2,63 @@
 
 {% block title %}Types de missions{% endblock %}
 
+{% block javascripts %}
+    {{ parent() }}
+    {{ encore_entry_script_tags('mission-type') }}
+{% endblock %}
+
 {% block body %}
     <h1>Types de missions</h1>
 
-    <table class="table mt-5">
-        <thead>
+    <div class="row">
+        <div class="col-12 text-right mb-2">
+            <a class="btn btn-success" href="{{ path('app_organization_mission_type_new') }}" role="button">Ajouter un nouveau type de mission</a>
```

Comment:
translation

Reply:
Yes, i will check in all modified templates if there are another missing translations